app/frontend_messages.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from flask.ext.babel import gettext as _
+
+logger = logging.getLogger(__name__)
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
+                'INSTRUCTIONS': _("Close this window to continue with your current survey."),
+                },
+            }
+
+
+def get_messages(system_message):
```

Comment:
`system_message` is only valid for our current use case, as this has been developed as a generic message loader, this would be more appropriately named something like `message_identifier`